UK-SBCoA/uniform-data-set-dotnet-web
# Maintain disabled state in RMREAS, RMMODE, NOT on postback

Fixes #220 

This update addresses an issue where the RMREAS, RMMODE, and NOT properties were incorrectly enabled when a form was submitted as "Finalized." To resolve this, a CSS class named never-enabled was added. This class overrides the valid class applied by jQuery validation during an HTTP POST request.

To test this fix:
- [x] Submit a form containing validation errors.
- [x] When the page reloads with errors displayed, ensure that the RMREAS, RMMODE, and NOT properties remain in their disabled state.


![image](https://github.com/user-attachments/assets/d5e10b36-f51b-45cd-94c5-6f5e9124b071)

All forms use this partial, so multiple forms will need to be tested during review to ensure consistent behavior.
